spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/ConstantColumnVector.java

```diff
@@ -20,19 +20,25 @@
 
 import org.apache.iceberg.spark.SparkSchemaUtil;
 import org.apache.iceberg.types.Type;
+import org.apache.iceberg.types.Types;
+import org.apache.spark.sql.catalyst.InternalRow;
+import org.apache.spark.sql.types.DataType;
 import org.apache.spark.sql.types.Decimal;
+import org.apache.spark.sql.types.StructType;
 import org.apache.spark.sql.vectorized.ColumnVector;
 import org.apache.spark.sql.vectorized.ColumnarArray;
 import org.apache.spark.sql.vectorized.ColumnarMap;
 import org.apache.spark.unsafe.types.UTF8String;
 
 class ConstantColumnVector extends ColumnVector {
 
+  private final Type icebergType;
   private final Object constant;
   private final int batchSize;
 
-  ConstantColumnVector(Type type, int batchSize, Object constant) {
```

Comment:
I renamed `type` to `icebergType` cause the parent class already provide `type` variable but it is a Spark type.